package.json

```diff
@@ -87,6 +94,7 @@
     "lodash": "^4.17.20",
     "merge": "^1.2.1",
     "mime-db": "^1.40.0",
+    "mongoose": "^6.2.6",
```

Comment:
Remove `mongoose` package from here as the package is not being used

Reply:
Check https://github.com/internxt/bridge/pull/75